modules/modal/react/lib/ModalContent.tsx

```diff
@@ -172,7 +176,28 @@ const ModalContent = ({
     }
   };
 
-  return (
+  React.useEffect(() => {
+    const siblings = [...((document.body.children as any) as HTMLElement[])].filter(
+      el => el !== modalRef.current!.parentElement
```

Comment:
I wonder if we should actually check if current is not undefined?
```
  const observed = useRef(null);

  useEffect(() => {
     if(observed.current) console.log(observed.current);
  }, [observed]);
```

Reply:
Interesting... Will `useEffect` ever be called before the ref is applied? Technically this is an unsupported use-case. If this case happens, we cannot guarantee accessibility of the Modal